builder-20/builder.toml

```
@@ -43,7 +43,7 @@ version = "0.20.8"
 
 [[buildpacks]]
   id = "heroku/python"
-  uri = "docker://docker.io/heroku/buildpack-python@sha256:4054a84d88d9a37d3798df5e7e33ea4d817d54fd56a8806a493bc47588ef24d6"
+  uri = "docker://docker.io/heroku/buildpack-python@sha256:c74c82e5b0bd94117370c71352a6ae93f293e8a3d75531b983126c0ecbd2f691"
```

The Python CNB has dropped support for the EOL Heroku-20, so the new version can't be used in `heroku/builder:20`. I'll manually revert the automation changes to this manifest short term, until the other languages also drop support, at which point we can update the list of builders the automation edits, here:
https://github.com/heroku/languages-github-actions/blob/b969dafe15a746b8d8fc7c16fef1a78dfee104d0/.github/workflows/_buildpacks-release.yml#L427

```suggestion
  uri = "docker://docker.io/heroku/buildpack-python@sha256:4054a84d88d9a37d3798df5e7e33ea4d817d54fd56a8806a493bc47588ef24d6"
```